libs/ledger-live-common/src/bot/specs.ts

```diff
@@ -119,7 +120,12 @@ export function deviceActionFlow<T extends Transaction>(
 
     if (!finalState) {
       let possibleKnownStep: Step<T> | null | undefined =
-        description.steps.find((s) => event.text.startsWith(s.title));
+        description.steps.find((s) => {
+          if (s.maxY) {
```

Comment:
if maxY is `0` this would be falsy. I recommend to use `typeof X === "number"`

Reply:
@henrily-ledger are you OK with that?